src/io/flutter/logging/v2/FlutterLogView.kt

```diff
@@ -0,0 +1,141 @@
+    if (lastEntryFilter != null && lastEntryFilter == filter) {
+      return
+    }
+    lastEntryFilter = filter
+    myConsoleView.clear()
+    val filterdEntries = if (filter == null) flutterLog.entries else flutterLog.entries.filter { filter.accept(it) }
+    filterdEntries.forEach { myConsoleView.print(flutterLogFormater.format(it), ConsoleViewContentType.NORMAL_OUTPUT) }
+  }
+
+  override fun setOutputPaused(value: Boolean) {
+    myConsoleView.isOutputPaused = value
+  }
+
+  override fun attachToProcess(processHandler: ProcessHandler?) {
+    processHandler?.let { flutterLog.listenToProcess(it, this) }
+  }
+
+  override fun print(text: String, contentType: ConsoleViewContentType) = flutterLog.addConsoleEntry(text, contentType)
+
+  override fun hasDeferredOutput(): Boolean = myConsoleView.hasDeferredOutput()
+  override fun clear() = myConsoleView.clear()
+  override fun setHelpId(helpId: String) = myConsoleView.setHelpId(helpId)
+  override fun getContentSize(): Int = myConsoleView.contentSize
+  override fun createConsoleActions(): Array<AnAction> = myConsoleView.createConsoleActions()
+  override fun getComponent(): JComponent = simpleToolWindowPanel
+  override fun performWhenNoDeferredOutput(runnable: Runnable) = myConsoleView.performWhenNoDeferredOutput(runnable)
+  override fun getPreferredFocusableComponent(): JComponent = myConsoleView.preferredFocusableComponent
+  override fun isOutputPaused(): Boolean = myConsoleView.isOutputPaused
+  override fun addMessageFilter(filter: Filter) = myConsoleView.addMessageFilter(filter)
+  override fun printHyperlink(hyperlinkText: String, info: HyperlinkInfo?) = Unit
+  override fun canPause(): Boolean = myConsoleView.canPause()
+  override fun allowHeavyFilters() = myConsoleView.allowHeavyFilters()
+  override fun dispose() = myConsoleView.dispose()
+  override fun scrollTo(offset: Int) = myConsoleView.scrollTo(offset)
+
+
+  private fun createToolbar(): DefaultActionGroup = DefaultActionGroup().apply {
+    add(ConfigureAction())
+  }
+
+  private inner class ConfigureAction : AnAction("Configure", null, AllIcons.General.Gear), RightAlignedToolbarAction {
+    override fun update(event: AnActionEvent) {
+      event.presentation.isEnabled = flutterApp.isSessionActive()
+    }
+
+    override fun actionPerformed(event: AnActionEvent) {
+      val component = (event.presentation.getClientProperty("button") as? JComponent)
+          ?: (event.inputEvent.source as? JComponent)
+          ?: return
+
+      val popupMenu = ActionManager.getInstance().createActionPopupMenu(
+        ActionPlaces.UNKNOWN,
+        createPopupActionGroup()
+      )
+      popupMenu.component.show(component, component.width, 0)
+    }
+
+    private fun createPopupActionGroup(): DefaultActionGroup = DefaultActionGroup().apply {
+      add(ShowTimeStampsAction())
+      add(ShowLevelAction())
```

Comment:
I just implement those two actions
ShowTimeStampsAction & ShowLevelAction

I'm not sure, do we need to implement log line number.
The line-number of the log is relative (especially when we filtering).